android/gradle.properties

```diff
@@ -3,7 +3,7 @@ propMinSdkVersion=15
 # https://code.google.com/p/android/issues/detail?id=184567
 propTargetSdkVersion=22
 propBuildToolsVersion=22.0.1
-propVersionCode=600
+propVersionCode=601
 propVersionName=6.0
```

Comment:
Это зачем? Пока что у нас в проде версия 6.0, пусть и тут будет та же, изменений-то нет ещё.

> On Apr 1, 2016, at 2:54 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/gradle.properties <https://github.com/mapsme/omim/pull/2676#discussion_r58195442>:
> 
> > @@ -3,7 +3,7 @@ propMinSdkVersion=15
> >  # https://code.google.com/p/android/issues/detail?id=184567
> >  propTargetSdkVersion=22
> >  propBuildToolsVersion=22.0.1
> > -propVersionCode=600
> > +propVersionCode=601
> >  propVersionName=6.0
> А вот в мастере-то как раз стоит поменять на 6.0.1
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2676/files/8ce84f940a2f9d1ede4bed28671fd8ce87b9e86b#r58195442>